packages/client/hmi-client/src/services/charts.ts

```diff
@@ -10,7 +10,7 @@ import type { FunmanBox, FunmanConstraintsResponse } from './models/funman-servi
 const VEGALITE_SCHEMA = 'https://vega.github.io/schema/vega-lite/v5.json';
 
 export const CATEGORICAL_SCHEME = ['#1B8073', '#6495E8', '#8F69B9', '#D67DBF', '#E18547', '#D2C446', '#84594D'];
-
+export const SENSITIVITY_COLOUR_SCHEME = ['#4575B4', '#91BFDB', '#E0F3F8', '#FFFFBF', '#FEE090', '#FC8D59', '#D73027'];
```

Comment:
Would be good to have nature of the colour scheme either commented or in the name itself, eg sequential, diverging, categorical, ordinal ... etc. Since most folks cannot read hex-values very well.

Reply:
I've added a comment here to address this (I've also learned what a diverging colour scheme is 😄 )